test/Entropy.FunctionalTests/MvcTests/LocalizationSampleTest.cs

```diff
@@ -142,3 +143,7 @@ public static IEnumerable<object[]> LocalizationResourceData
         }
     }
 }
+#elif NETCOREAPP2_0
+#else
```

Comment:
The test was like this before the net46 target was removed. This change is just restoring the removed code, and changing the condition from NET46 to NET461.